src/pkg/brimcap/index.ts

```diff
@@ -0,0 +1,5 @@
+function load() {}
```

Comment:
Is this just some scaffolding from our convos about the brimcap loaders? 🤔

Reply:
You are correct. Will delete.